addons/calendar/__manifest__.py

```diff
@@ -48,14 +48,11 @@
         # Unit test files
         'web.assets_unit_tests': [
             'calendar/static/tests/**/*.js',
-            ('remove', 'calendar/static/tests/legacy/**/*'),  # to remove when all legacy tests are ported
             ('remove', 'calendar/static/tests/helpers/**/*'),
             ('remove', 'calendar/static/tests/tours/**/*'),
         ],
         'web.qunit_suite_tests': [
-            'calendar/static/tests/legacy/**/*',
             'calendar/static/tests/helpers/**/*',
```

Comment:
Are you sure helpers are still needed in qunit?

Reply:
Yes. `google_calendar` and `microsoft_calendar` still needs it.